main.py

```diff
@@ -1,12 +1,253 @@
-import requests
+# import requests
+from video_list import VideoList
+from customer_list import CustomerList
+from rental_list import RentalList
+
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
+
 def main():
     print("WELCOME TO RETRO VIDEO STORE")
     pass
 
-
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
+
+
+def print_stars():
+    print("\n**************************\n")
+
+
+def make_choice():
+
+    options = {
+        "1": "List all videos",
+        "2": "Add a video", 
+        "3": "Select a video",
+        "4": "Update selected video", 
+        "5": "Delete selected video", 
+        "6": "List all customers",
+        "7": "Add a customer",
+        "8": "Select a customer",
+        "9": "Update selected customer",
+        "10": "Delete selected customer",
+        "11": "Check-out video to customer",
+        "12": "Check-in video to customer",
+        }
+
+    print_stars()
+    print("Welcome to Michelle's Retro Video Store CLI")
+    print("Please select the action you would like to perform.")
+    print_stars()
+    
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_stars()
+
+    choice = input("Make your selection using the option number: ")
+    
+    return choice
+
+
+def select_video(video_list):
+    select_by = input("What would you like to search videos by?  Enter either title or id to begin: ")
+
+    if select_by=="title":
+        title = input("Please enter the video title you are looking for: ")
+        video = video_list.get_video(title=title)
+    elif select_by=="id":
+        id = int(input("Please enter the video id you are looking for: "))
+        video = video_list.get_video(id=id)
+    else:
+        print("Please enter id or title to specify search.")
+
```

Comment:
We need to add a `return` in this else clause, otherwise the code raises an exception on line 67 because the video has not been assigned. Consider returning `false`. This would work well with the rest of your logic.